coding/cpp_style_guide.rst

```diff
@@ -2079,87 +2087,88 @@ Example 2 adds an additional line without significant increase in readability.
 
 .. code-block:: cpp
 
-   class SomeClass :
-       public  BaseClass1,
-       public  BaseClass2,
-       private BaseClass3
-   {
+   class SomeClass : public BaseClass1, public BaseClass2, private BaseClass3 {
    public:
        SomeClass() {}
+   
    protected:
        ...
+   
    private:
        ...
    };
 
 Note that:
 
-- Like functions, the opening brace is at the same indentation as the class keyword
-- any subclasses are indented one block and their visibility is explicitly qualified
-- The declarations public, protected and private are left-justified to the same indentation as class.
-- If ``SomeClass`` is not derived from anything, the opening brace should still line up with class
-
-For historical reasons, the format:
-
-.. code-block:: cpp
-
-   class SomeClass : public BaseClass1, BaseClass2, private BaseClass3 {
-   public:
-       SomeClass() {}
-   protected:
-       ...
-   private:
-       ...
-   };
-
-is permitted but not encouraged, and should not be used in new code.
+  - Any base class name should be on the same line as the subclass name,
```

Comment:
Make sure you note when publicizing this on e.g. community.lsst.org that this is a change from previously-permitted to required.